src/routes/events/components/Events.js

```diff
@@ -51,6 +60,31 @@ export default class Events extends React.Component<PropsType> {
     navigateToEvent({cityCode, language, path, forceUpdate: true, key: navigation.getParam('key')})
   }
 
+  createNavigateToFeedbackForEvent = (event: EventModel) => (isPositiveFeedback: boolean) => {
+    const {t, navigation, cities, cityCode} = this.props
+    const cityTitle = CityModel.findCityName(cities, cityCode)
+    navigation.navigate('FeedbackModal', {
+      isPositiveFeedback,
+      feedbackItems: [
+        new FeedbackVariant(t('feedback:contentOfPage', {page: event.title}), PAGE_FEEDBACK_TYPE),
+        new FeedbackVariant(t('feedback:contentOfCity', {city: cityTitle}), PAGE_FEEDBACK_TYPE),
+        new FeedbackVariant(t('feedback:technicalTopics'), CATEGORIES_FEEDBACK_TYPE)
+      ]
+    })
+  }
+
+  navigateToFeedbackForEvents = (isPositiveFeedback: boolean) => {
+    const {t, navigation, cities, cityCode} = this.props
+    const cityTitle = CityModel.findCityName(cities, cityCode)
+    navigation.navigate('FeedbackModal', {
+      isPositiveFeedback,
+      feedbackItems: [
+        new FeedbackVariant(t('feedback:contentOfCity', {city: cityTitle}), PAGE_FEEDBACK_TYPE),
+        new FeedbackVariant(t('feedback:technicalTopics'), CATEGORIES_FEEDBACK_TYPE)
```

Comment:
Couldn't you extract this to a helper? We need these two FeedbackVariants quite a lot already, and it's just duplicated code.  `(props)  => [FeedbackItems]`

Reply:
I will look into this in NATIVE-208 once again -- for now I'm not yet sure if it makes sense...